src/js/puzzles.js

```diff
@@ -335,6 +335,27 @@ var puzzlesArray = [
   word: 'sun',
   hint: 'Time of my life',
   answer: 'fun'
+},
+{
+  id: 49,
+  sentence: 'The star is out of my peach.',
+  word: 'peach',
+  hint: 'beyond the limit',
+  answer: 'reach'
+},
+{
+  id: 50,
+  sentence: 'Ferry pie makes for an amazing treat.',
+  word: 'ferry',
+  hint: 'small, red fruit',
+  answer: 'cherry'
+},
+{
+  id: 51,
+  sentence: 'I scheme to take over the world.',
```

Comment:
Because this sentence makes sense, it may be difficult for someone to guess. Perhaps consider something that makes less sense, like `bream` or `beam`.

Reply:
Updated as requested.

Reply:
Also, I accidentally clicked "Resolve Conversation".